pkg/util/mount/mount_windows_test.go

```diff
@@ -551,3 +552,46 @@ func TestPathWithinBase(t *testing.T) {
 			test.fullPath, test.basePath, result, test.expectedResult)
 	}
 }
+
+func TestGetFileType(t *testing.T) {
+	mounter := New("fake/path")
+
+	tempDir, err := ioutil.TempDir("", "test-get-filetype")
+	if err != nil {
+		t.Fatalf("unexpected error: +%v when creating temp directory", err)
+	}
+	t.Log(tempDir)
+	defer os.RemoveAll(tempDir)
+
+	tempFile, err := ioutil.TempFile("", "test-get-filetype")
+	if err != nil {
+		t.Fatalf("unexpected error: +%v when creating temp file", err)
+	}
+	t.Log(tempFile.Name())
+	defer tempFile.Close()
+	defer os.RemoveAll(tempFile.Name())
+
+	testCase := []struct {
+		filePath     string
+		expectedType FileType
```

Comment:
Are the other file types supported in Windows? cc @andyzhangx

Reply:
@msau42 good question: I have checked the mount_windows [code](https://github.com/kubernetes/kubernetes/blob/master/pkg/util/mount/mount_windows.go#L96), currently only Directory is supported, I will make another PR to support File type. So for this test, it's ok only to cover Directory and File type now.